js/js-array-methods/js-array-methods_filter/index.js

```diff
@@ -23,13 +23,25 @@ const cards = [
   },
 ];
 
-const onlyCardWithIdTwo = null;
+const onlyCardWithIdTwo = cards.filter((card) => card.id === "2");
 
-const allCardsWith3Tags = null;
+const allCardsWith3Tags = cards.filter((card) => card.tags.length === 3);
 
-const allCardsThatAreNotBookmarked = null;
+const allCardsThatAreNotBookmarked = cards.filter(
+  (card) => card.isBookmarked === false
+);
 
-const allCardsWithTagsHTMLOrJSThatAreBookmarked = null;
+const allCardsWithTagsHTMLOrJSThatAreBookmarked = cards
+  .filter((card) => card.isBookmarked)
+  .filter((card) => card.tags.includes("html") || card.tags.includes("js"));
```

Comment:
this line somehow didn't work for me but as shown in the class it's correct. looks good!